src/Criteo.OpenApi.Comparator.UTest/Resource/issue_49_false_1015_out_of_order_parameters/diff.json

```diff
@@ -0,0 +1,35 @@
+[
```

Comment:
This test shows that when two parameters and out of order, 1042 issues are still detected. This is the equal case, where the schemas differ in parameter order.